src/services/webclient.ts

```diff
@@ -1120,6 +1126,9 @@ export class WebClientService {
             this.ackTimer = null;
         }
 
+        // Stop other timers
+        this.timeoutService.cancelAll();
```

Comment:
This should only be done in case `close` is `true` (move it down). Furthermore, this also needs to be done when discarding a session after having tried to resume it.

TL;DR: Move this directly beside calls to `this.wireMessageFutures.clear()`.